lib/taskana-core/src/main/java/pro/taskana/impl/SummaryServiceImpl.java

```diff
@@ -28,17 +29,23 @@ public SummaryServiceImpl(TaskanaEngine taskanaEngine, SummaryMapper summaryMapp
 
     @Override
     public List<TaskSummary> getTaskSummariesByWorkbasketId(String workbasketId) {
+        LOGGER.debug("entry to getTaskSummariesByWorkbasketId(workbasketId = {}", workbasketId);
         List<TaskSummary> taskSummaries = new ArrayList<>();
         try {
             taskanaEngineImpl.openConnection();
             taskSummaries = summaryMapper.findTasksummariesByWorkbasketId(workbasketId);
         } catch (Exception ex) {
-            LOGGER.error("Getting TASKSUMMARY failed internal.", ex);
+            LOGGER.warn("Getting TASKSUMMARY failed internally.", ex);
         } finally {
             if (taskSummaries == null) {
```

Comment:
wofür ist das gut? Bei einer Exception möchte ich diese auch weitergeben. Ggf. in eine TaskanaException verpackt.